capstone_lesson6_v3.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+print("A third way to solve: ")
+
+#step 1: get input from user for total number of minutes as an integer
+total_minutes = int(input("What is the total number of minutes? "))
+print(total_minutes)
+
+#step 2: convert minutes to hours
+total_hours = int(total_minutes / 60)
+# print(total_hours)
+
+#step 3: calculate remainder and turn into minutes
+leftover_minutes = round(total_minutes % 60)
```

Comment:
Ok, I see your point about rounding. As for the constant, I want to make sure I understand your comment. 

So instead I'd declare a constant at the start of the code (ie, MINUTES_PER_HOUR = 60) and use this instead of 60 in this line of code? It'd read: leftover_minutes = total_minutes % MINUTES_PER_HOUR

Reply:
Exactly! 💯